laborator/content/introducere-asamblare/solution/7-cf-of/cf.asm

```diff
@@ -0,0 +1,22 @@
+%include "printf32.asm"
+
+section .text
+    global main
+    extern printf
+
+main:
+    xor al, al
+    mov al, 0xFF
+    
+no_flag1:
+    PRINTF32 `CF nu e activ\n\x0`
+    test al, al
+    add al, 1
```

Comment:
This is the same example as `cf-of.asm`. Use something else, such as `200 + 100` to highlight the difference.